stdlib/public/core/Misc.swift

```diff
@@ -159,6 +159,17 @@ public func _unsafePerformance<T>(_ c: () -> T) -> T {
   return c()
 }
 
+// Helper function that exploits a bug in rethrows checking to
+// allow us to call rethrows functions from generic typed-throws functions
+// and vice-versa.
+@usableFromInline
+@_alwaysEmitIntoClient
+@inline(__always)
+func _rethrowsViaClosure(_ fn: () throws -> ()) rethrows {
```

Comment:
This is no longer used in the current version of the PR

Reply:
Ooohhh, you're right! Thank you